lib/queue.js

```diff
@@ -114,6 +115,7 @@ var Queue = function Queue(name, redisPort, redisHost, redisOptions){
   // Create queue client (used to add jobs, pause queues, etc);
   //
   this.client = createClient();
+  this.rateLimitClient = createClient();
 
```

Comment:
Could we reuse a redis client? people were already complaining that 3 redis clients per queue instance is too much (since many uses bull with redis as a service and pay by client).